tests/nfs2-frames/test.yaml

```diff
@@ -0,0 +1,56 @@
+requires:
+  min-version: 7.0
+  files:
+      - rust/src/nfs/nfs2.rs
```

Comment:
I think we don't need to use that any longer, these days for Rust is not optional.